src/test/com/abstract_calss/Horse.java

```diff
@@ -0,0 +1,12 @@
+package test.com.abstract_calss;
+
+public class Horse extends Animal {
+    @Override
+    void move() {
+        System.out.println("run");
+    }
+    @Override
+    void talk() {
+
```

Comment:
Google it, you can rename it to "makeSound". If you don't know what to put somewhere you shouldn't just leave it empty. You have a task and you need to finish it, if you don't understand what should be done from the description you should ask, those tasks are primitive examples of your work and other people will spend time reviewing your work and when your work tells them that you don't care about the result of your work but your goal to finish it fast even when it means cutting corners and leaving some parts empty, nobody will be happy working with you.